app/src/main/java/fi/thl/koronahaavi/service/GoogleExposureNotificationService.kt

```diff
@@ -78,6 +80,11 @@ class GoogleExposureNotificationService(
 
     override fun deviceSupportsLocationlessScanning() = client.deviceSupportsLocationlessScanning()
 
+    private fun isOwnerUserProfile(): Boolean =
+        (context.getSystemService(Context.USER_SERVICE) as? UserManager)?.let {
+            it.getSerialNumberForUser(Process.myUserHandle()) == 0L
```

Comment:
Maybe add a comment regarding the origin of the ==0 check (I couldn't find a reference from UserManager).

Reply:
Yes good point, comment added